examples/cryptocurrency/src/lib.rs

```diff
@@ -153,6 +155,8 @@ pub mod transactions {
 
 /// Contract errors.
 pub mod errors {
+    #![allow(bare_trait_objects)]
```

Comment:
Maybe it would be better to leave s small explanation comment why this is allowed here (and with other allows)? With the link to ecr-1771?

Reply:
Fixed. 👍